SPECS/ocaml-fileutils/ocaml-fileutils.spec

```diff
@@ -35,57 +34,40 @@ abstract filenames.
 
 %package        devel
 Summary:        Development files for %{name}
-Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
 
 %description    devel
 The %{name}-devel package contains libraries and signature files for
 developing applications that use %{name}.
 
 %prep
-%setup -q
-%setup -q -T -a 1 -D
-%autopatch -p1
+%autosetup -n %{name}-%{version} -p1
 
 %build
-ocaml setup.ml -configure --prefix %{_prefix} --destdir %{buildroot} \
-	--enable-tests
-make
+%dune_build
 
 %install
-export DESTDIR=%{buildroot}
-export OCAMLFIND_DESTDIR=%{buildroot}%{_libdir}/ocaml
-mkdir -p $OCAMLFIND_DESTDIR $OCAMLFIND_DESTDIR/stublibs
-
-# Set htmldir to current directory, then copy the docs (in api/)
-# as a %doc rule.
-make htmldir=. install
+%dune_install
 
+# Do not run the tests (RHEL 7+ only) since they require ocaml-ounit.
+%if 0%{?fedora} || 0%{?rhel} || %{?azl} <= 6
```

Comment:
This doesn't look right. Was the `<= 6` part of `0%{?rhel} <= 6`?

Reply:
Also, please do `0%{?azl}` instead of plain `%{?azl}`. If `azl` is not set, then you can end up with an empty `%if` and I think `rpmbuild` will crash.

Reply:
Ah, I I missed it. I fixed it in above